projects/taiga-schematics/schematics/ng-add/steps/add-taiga-styles.ts

```diff
@@ -29,7 +29,7 @@ export function addTaigaStylesToAngularJson(
     if (!styles) {
         targetOptions.styles = taigaStyles;
     } else {
-        targetOptions.styles = [...styles, ...taigaStyles];
+        targetOptions.styles = Array.from(new Set([...styles, ...taigaStyles]));
     }
```

Comment:
Put Taiga styles before, in case people will want to override something.

Reply:
can we write tests for this case?

Reply:
done ✅